### DISCUSSION TITLE
Request to get a release in NRO format

### DISCUSSION THREAD

**Discussion by JeffreyO** [xerpi/vita2hos#35] [Q&A] [answered]
I'm trying to run this using the Skyline emulator, but it complains about the NSP file being invalid and doesn't even attempt to load it. It does load NRO files correctly, though.

Would it even be possible to use Skyline with this?

**Comment by TSRBerry** [comment, marked as answer]
I don't know about skyline, but currently xerpi can't offer a NRO release, since the hb-launcher currently only supports 64-bit homebrew.
